src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let stateCopy = { ...state };
+  const stateCopies = [];
```

Comment:
I think you should not name variables with a difference of several letters. Because of this, there may be errors in the future.